src/index.js

```diff
@@ -2,5 +2,69 @@ import bordList from './modules/bordList.js';
 import form from './modules/form.js';
 import './style.css';
 
-bordList();
-form();
+const URL = 'https://us-central1-js-capstone-backend.cloudfunctions.net/api';
+
+let api = JSON.parse(localStorage.getItem('api')) || '';
+
+const getGameId = async () => {
+  const response = await fetch(`${URL}/games/`, {
+    method: 'POST',
+    headers: {
+      'Content-Type': 'application/json',
+    },
+    body: JSON.stringify({ name: "Phone Khaing Hein's games" }),
+  });
+
+  const { result } = await response.json();
+  localStorage.setItem('api', JSON.stringify(result.split(' ')[3]));
+};
+
+if (api === '') {
+  getGameId();
+  api = JSON.parse(localStorage.getItem('api')) || '';
+}
+
+const getScores = async () => {
+  const response = await fetch(`${URL}/games/${api}/scores/`);
+
+  const { result } = await response.json();
+  localStorage.setItem('scores', JSON.stringify(result));
+};
+
+const init = () => {
+  getScores();
+  bordList();
+  form();
+};
+
+init();
+
+const formElement = document.getElementById('scoreForm');
+formElement.addEventListener('submit', async (e) => {
+  e.preventDefault();
+  const data = {
+    user: formElement.name.value.trim(),
+    score: formElement.score.value.trim(),
+  };
+
+  if (data.user !== '' && data.score !== '') {
+    const response = await fetch(`${URL}/games/${api}/scores`, {
+      method: 'POST',
+      headers: {
+        'Content-Type': 'application/json',
+      },
+      body: JSON.stringify(data),
+    }).then((response) => {
+      const json = response.json();
+      document.getElementById('board').innerHTML = '';
+      init();
+    });
+  }
+});
+
+const refresh = document.getElementById('refresh');
+
+refresh.addEventListener('click', () => {
+  document.getElementById('board').innerHTML = '';
+  init();
+});
```

Comment:
Nice work overall 💯 but for some reason, the `Refresh` button doesn't work right after I submit the `POST` request (I even tried with an `alert` but it doesn't work, so the issue is not API-related), it works before but not after, in order to get the scores I have to refresh manually. 👀  Kindly fix that as the main purpose of the refresh button is to update the list to its latest version. 🤓 

PS: Read the other comment where I explain why I think it doesn't work  📖